resources/views/admin/users/index.blade.php

```diff
@@ -30,105 +30,175 @@
+                    <vue-password v-model="addUser.password" :disable-toggle=true ref="passwordStrength">	
+                        <div slot="password-input" slot-scope="props">	
+                            {!!Form::password('password', ['class'=> 'form-control', 'v-model'=> 'addUser.password',	
+                            '@input' => 'props.updatePassword($event.target.value)', 'autocomplete' => 'new-password',	
+                            'v-bind:class' => '{\'form-control\':true, \'is-invalid\':addUser.addError.password}'])!!}	
+                        </div>	
+                    </vue-password>	
+                </div>	
+
+                <div class="form-group">	
+                    {!!Form::label('confpassword', __('Confirm Password'))!!}<small class="ml-1">*</small>	
+                    {!!Form::password('confpassword', ['class'=> 'form-control', 'v-model'=> 'addUser.confpassword',	
+                    'v-bind:class' => '{\'form-control\':true, \'is-invalid\':addUser.addError.password}', 'autocomplete' => 'new-password'])!!}	
+                    <div class="invalid-feedback" v-for="password in addUser.addError.password">@{{password}}</div>	
+                </div>
+            </b-modal>
+
+            {{-- <div class="modal" tabindex="-10" role="dialog" id="restoreUser">
+                <div class="modal-dialog modal-dialog-centered" rold="document">
+                    <div class="modal-content">
+                        <div class="modal-header">
+                            <h5 class="modal-title">{{ __('Deleted User Found') }}</h5>
+                            <button type="button" class="close" data-dismiss="modal" aria-label="Close">	
+                                <span aria-hidden="true">&times;</span>	
+                            </button>
+                        </div>
+
+                        <div class="modal-body">
+                            <p>{{ __('An existing user has been found with the ')}} @{{ restoreAttribute + ' "' + restoreValue + '"' }} {{__(' would you like to save and reactivate their account?') }}</p>
+                        </div>
+
+                        <div class="modal-footer">
+                            <button type="button" class="btn btn-outline-secondary" @click="onCancelRestore">	
+                                {{ __('Cancel') }}
+                            </button>
+
+                            <button type="button" class="btn btn-secondary ml-2" @click="onSaveRestore">	
+                                {{ __('Save') }}
+                            </button>
+                        </div>
                     </div>
-                </template>
-            </add-user-modal>
+                </div>
+            </div> --}}
         @endcan
     </div>
 @endsection
 
 @section('js')
     <script src="{{mix('js/admin/users/index.js')}}"></script>
+
+    @can('create-users')	
+    <script>	
+    
+    // var restoreUserModal = new Vue({	
```

Comment:
Don't comment out code. We can look at the DIFF if needed. Also, please move this back into the blade for ease of maintenance.